tests/cypress/integration/actions_tasks_objects/case_27_undo_redo_feature.js

```diff
@@ -0,0 +1,83 @@
+// Copyright (C) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName, labelName } from '../../support/const';
+
+context('Undo/redo feature', () => {
+    const caseId = '27';
+    const firstRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 100,
+        firstY: 100,
+        secondX: 150,
+        secondY: 150,
+    };
+    const secondRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 150,
+        firstY: 150,
+        secondX: 200,
+        secondY: 200,
+    };
+    const thirdRectangleShape2Points = {
+        points: 'By 2 Points',
+        type: 'Shape',
+        labelName: labelName,
+        firstX: 200,
+        firstY: 200,
+        secondX: 250,
+        secondY: 250,
+    };
+
+    function checkExistObject(stateFirstObject, stateSecondObject, stateThirdObject){
+        // check objects on background
+        cy.get('#cvat_canvas_shape_1').should(stateFirstObject);
+        cy.get('#cvat_canvas_shape_2').should(stateSecondObject);
+        cy.get('#cvat_canvas_shape_3').should(stateThirdObject);
+        // check objects on sidebar
+        cy.get('#cvat-objects-sidebar-state-item-1').should(stateFirstObject);
+        cy.get('#cvat-objects-sidebar-state-item-2').should(stateSecondObject);
+        cy.get('#cvat-objects-sidebar-state-item-3').should(stateThirdObject);
+    };
+
+    before(() => {
+        cy.openTaskJob(taskName);
+    });
+
+    describe(`Testing case "${caseId}"`, () => {
+        it('Create objects', () => {
+            cy.createRectangle(firstRectangleShape2Points);
+            cy.createRectangle(secondRectangleShape2Points);
+            cy.createRectangle(thirdRectangleShape2Points);
```

Comment:
Should not such the things be in ``before`` callback? I suppose they are preparation steps since we do not testing here creating of rectangles.

Reply:
Good remark. Step for create objects moved to before function.